src/travel-search-filters/FiltersContext.tsx

```diff
@@ -0,0 +1,51 @@
+import React, {createContext, useContext, useEffect, useState} from 'react';
+import {storedFilters} from './storage';
+import {TransportModeFilterOptionWithSelectionType} from '@atb/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/types';
+
+type FiltersContextState = {
+  filters: TransportModeFilterOptionWithSelectionType[];
+  setFilters(
+    filters: TransportModeFilterOptionWithSelectionType[] | undefined,
+  ): Promise<void>;
+};
+const FiltersContext = createContext<FiltersContextState | undefined>(
+  undefined,
+);
+
+const FiltersContextProvider: React.FC = ({children}) => {
+  const [filters, setFiltersState] = useState<
+    TransportModeFilterOptionWithSelectionType[]
+  >([]);
+  async function populateFilters() {
+    const [filters] = await Promise.all([storedFilters.getFilters()]);
+    setFiltersState(filters ?? []);
+  }
+
+  useEffect(() => {
+    populateFilters();
```

Comment:
Seems like `populateFilters` is only used here? In that case I think it is better to inline it. In general `useEffect` functions which runs only on first render shouldn't use reactive values.